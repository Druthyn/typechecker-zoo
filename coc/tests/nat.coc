inductive Nat : Type with
  | zero : Nat
  | succ : Nat -> Nat