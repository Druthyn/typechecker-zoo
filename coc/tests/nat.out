Parsed module with 1 declarations
âœ“ Module 'tests/nat.coc' type checks successfully!